src/features/projectsV2/ProjectDetails/index.tsx

```diff
@@ -176,13 +188,18 @@ const ProjectDetails = ({
               hoveredPlantLocation?.type === 'multi-tree-registration'
                 ? hoveredPlantLocation
                 : selectedPlantLocation?.type === 'multi-tree-registration'
-                ? selectedPlantLocation
-                : undefined
+                  ? selectedPlantLocation
+                  : undefined
             }
             setSelectedSamplePlantLocation={setSelectedSamplePlantLocation}
             isMobile={isMobile}
           />
         )}
+
+        {shouldShowOtherIntervention && (selectedPlantLocation &&
+          selectedPlantLocation.type !== 'single-tree-registration' || hoveredPlantLocation && hoveredPlantLocation.type !== 'single-tree-registration') ? <OtherInterventionInfo plantLocationInfo={selectedPlantLocation || hoveredPlantLocation} setSelectedSamplePlantLocation={setSelectedSamplePlantLocation}
+            isMobile={isMobile} /> : null}
+
```

Comment:
Move this into the condition for `shouldShowOtherIntervention` to keep the JSX clean.